### PR TITLE
ISSUE-76 Ilya MIkhaylov

### PR DESCRIPTION
**Pull request by IlyaMikhaylovTim**
Solved first problem two times, also solved one more problem (fourth or third, don't remember exactly).

https://my.compscicenter.ru/users/5622/

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -1,4 +1,4 @@
-71 + 11 = 82
+71 + 11 = 94
```
Comment (lo0op) [inline tasks]: incorrect answer here

**Review by lo0op — CHANGES_REQUESTED** [review]

File: tasks (hunk 1 above)
Comment (IlyaMikhaylovTim) [reply tasks]: Поправил, чекай.

**Review by lo0op — APPROVED** [review]

**Comment by dehasi** [comment]
@IlyaMikhaylovTim `ISSUE-INFTY` -> `ISSUE-76`

**Review by dehasi — APPROVED** [review]

**Comment by dehasi** [comment]
@IlyaMikhaylovTim  Номер тикета в коммитах поправьте пожалуйста.

**Review by dehasi — APPROVED** [review]

**Pull request merged**